intro.tex

```diff
@@ -1,7 +1,10 @@
 \section{Introduction}
```

Comment:
I shuffled the phrasing around so it's clear from the beginning who the customers of this document are (meaning Chilean unionized employees and not AURA except)